README.md

```diff
@@ -0,0 +1,355 @@
+- Each database can contain many collections, which are similar to tables in a relational database.
+- Each collection can then contain many documents. Think rows, just better!
+  Documents are super flexible, easy to read and easy to work with JSON-like structures that contain our data.
+
+## Creating some test data in Atlas
+
+Since there is no data yet, you will see an empty list of databases and collections.
+Click on `Add My Own Data` to add the first entry.
+
+![15_collections.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/15_collections_601bf59508.jpg "Collections overview for your Atlas cluster")
+
+The database name and collection name can be anything but to be in line with the code we'll see later,
+call them `crypto-news-website` and `news` respectively and hit `Create`.
+
+![16_create_database.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/16_create_database_afa2bb61d2.jpg "Create Database pop-up")
+
+This should lead to a new entry that looks like this:
+
+![17_your_new_collection.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/17_your_new_collection_25096bc341.jpg "Newly created database and collection.")
+
+Next, click on `INSERT DOCUMENT`.
+
+There are a couple things going on here. The `_id` has already been created automatically.
+Each document contains one of those and they are of type `ObjectId`. It uniquely identifies the document.
+
+By hovering over the line count on the left, you'll get a pop-op to add more fields.
+Add one called `title` and set its value to whatever you like, the screenshot shows an example you can use.
+Choose `String` as the type on the right.
+Next, add a `date` and choose `Date` as the type on the right.
+
+![18_insert_document.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/18_insert_document_c7ce90362e.jpg "Inserting a new document into the collection.")
+
+Repeat the above process a couple times to get as much example data in there as you like.
+You may also just continue with one entry though if you like and fill up your news when you are done.
+
+## Creating a connection string to access your MongoDB Atlas cluster
+
+The final step within MongoDB Atlas is to actually create access to this database so that the MongoDB Driver we
+installed into the project can connect to it.
+This is done by using a [connection string](https://www.mongodb.com/docs/manual/reference/connection-string/).
+A connection string is a URI that contains username, password and the host address of the database you want
+to connect to.
+
+Click on `Databases` on the left to get back to the cluster overview.
+
+This time, hit the `Connect` button and then `Connect Your Application`.
+If you haven't done so before, choose a username and password for the database user accessing this cluster during the
+tutorial.
+Also, add `0.0.0.0/0` as the IP address so that the Azure deployment can access the cluster later on.
+
+Copy the `connection string` that's shown in the pop-up.
```

Comment:
This could be a good time to link to the page explaining the various elements of a connection string
https://www.mongodb.com/docs/manual/reference/connection-string/

Reply:
Fixed with c66c3b5b8d5ac740653ca9fa0e23bd4ab67b0f8c